packages/forms/src/UIForm/fields/Enumeration/EnumerationWidget.js

```diff
@@ -0,0 +1,1043 @@
+				icon: 'talend-pencil',
+				id: 'edit',
+				onClick: this.onEnterEditModeItem.bind(this),
+			},
+			{
+				disabled: disabledAction,
+				label: t('ENUMERATION_WIDGET_REMOVE_VALUE', { defaultValue: 'Remove value' }),
+				icon: 'talend-trash',
+				id: 'delete',
+				onClick: this.onDeleteItem.bind(this),
+			},
+		];
+		this.defaultHeaderActions = [
+			{
+				disabled: false,
+				label: t('ENUMERATION_WIDGET_SEARCH_VALUES', {
+					defaultValue: 'Search for specific values',
+				}),
+				icon: 'talend-search',
+				id: 'search',
+				onClick: this.changeDisplayToSearchMode.bind(this),
+			},
+		];
+
+		if (this.allowImport) {
+			const dataFeature = this.props.schema['data-feature'];
+			this.defaultHeaderActions.push({
+				disabled: disabledAction,
+				label: t('ENUMERATION_WIDGET_IMPORT_FROM_FILE', {
+					defaultValue: 'Import values from a file',
+				}),
+				icon: 'talend-download',
+				id: 'upload',
+				onClick: this.onImportButtonClick.bind(this),
+				'data-feature': dataFeature ? dataFeature.importFile : undefined,
+				displayMode: 'dropdown',
+				items: [
+					{
+						label: t('ENUMERATION_WIDGET_ADD_FROM_FILE', {
+							defaultValue: 'Add values from a file',
+						}),
+						id: 'append-uploading',
+						onClick: this.onImportAppendClick.bind(this),
+						'data-feature': dataFeature ? dataFeature.addFromFile : undefined,
+					},
+					{
+						label: t('ENUMERATION_WIDGET_OVERWRITE_VALUES', {
+							defaultValue: 'Overwrite existing values',
+						}),
+						id: 'overwrite-uploading',
+						onClick: this.onImportOverwriteClick.bind(this),
+						'data-feature': dataFeature ? dataFeature.overwriteExisting : undefined,
+					},
+				],
+			});
+		}
+
+		this.defaultHeaderActions.push({
+			label: t('ENUMERATION_WIDGET_ADD_ITEM', { defaultValue: 'Add item' }),
+			icon: 'talend-plus',
+			id: 'add',
+			disabled: disabledAction,
+			onClick: this.changeDisplayToAddMode.bind(this),
+		});
+
+		this.selectedHeaderActions = [
+			{
+				disabled: disabledAction,
+				label: t('ENUMERATION_WIDGET_REMOVE_SELECTED_VALUES', {
+					defaultValue: 'Remove selected values',
+				}),
+				icon: 'talend-trash',
+				id: 'delete',
+				onClick: this.onDeleteItems.bind(this),
+			},
+		];
+
+		let defaultDisplayMode = enumerationStates.DISPLAY_MODE_DEFAULT;
+		if (props.schema && props.schema.displayMode) {
+			defaultDisplayMode = props.schema.displayMode;
+		}
+
+		this.state = {
+			inputRef: this.setInputRef.bind(this),
+			displayMode: defaultDisplayMode,
+			searchCriteria: '',
+			required: (props.schema && props.schema.required) || false,
+			headerDefault: this.defaultHeaderActions,
+			headerSelected: this.selectedHeaderActions,
+			headerInput: this.addInputs,
+			items: (props.value || []).map(item => ({
+				id: item.id,
+				values: item.values,
+			})),
+			itemsProp: {
+				key: 'values',
+				getItemHeight: this.constructor.getItemHeight.bind(this),
```

Comment:
This look like you want to do is either Enumeration.getItemHeight which just return a constant. isnt it ?